terraform/gcp/modules/external_secrets/external_secrets.tf

```diff
@@ -54,27 +66,6 @@ resource "google_service_account_key" "external_secrets_key" {
   service_account_id = google_service_account.external_secrets_sa.name
 }
 
-resource "kubernetes_secret_v1" "gcpsm_secret" {
-  metadata {
-    name        = "gcpsm-secret"
-    namespace   = "external-secrets"
-    annotations = {}
-    labels      = {}
-  }
-
-  binary_data = {
-    secret-access-credentials = google_service_account_key.external_secrets_key.private_key
-  }
-
-  type = "Opaque"
-
-  depends_on = [
-    google_service_account.external_secrets_sa,
-    google_service_account_key.external_secrets_key,
-    helm_release.external_secrets
-  ]
-}
-
 resource "kubectl_manifest" "secretstore_gcp_backend" {
   yaml_body = <<YAML
 apiVersion: external-secrets.io/v1alpha1
```

Comment:
maybe we can bump to `apiVersion: external-secrets.io/v1beta1` and use the new version

Reply:
we are still on 0.4.4, i dont think it supports v1beta1

though we should upgrade external secrets to 0.5x
filed https://github.com/sigstore/public-good-instance/issues/480

Reply:
i think we don't use anything that breaks, but i will double check